app/storybook/documentation/ui-kit/hooks/useTopValues.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/blocks'
+import LinkTo from '@storybook/addon-links/react'
+import { SourceCode } from '../../../.storybook/blocks/SourceCode'
+import * as useTopValuesStories from '../../../../../packages/ui-kit/src/hooks/useTopValues/useTopValues.stories'
+
+<Meta title="Hooks/useTopValues" />{' '}
+
+# `useTopValues`
+
+The `useTopValues` is a proposed Query Hook in the Propel’s UI Kit library. This Hook abstracts the code for querying
+the GraphQL API, allowing customers to implement their own components using the Hook. The `useTopValues` hook would return an
+array of the most frequent values in a given column. The resulting array is sorted in descending order of approximate
+frequency of values.
+
+<SourceCode of={useTopValuesStories.TopValuesChart} shown />
```

Comment:
I'm getting the following error in Storybook:
```
{
    "data": null,
    "errors": [
        {
            "message": "Principal APP01H3M5C2850FSDMDCN73VYGZF3 denied for User- or Application-only operation \"topValues\"",
            "extensions": {
                "code": "UNAUTHORIZED"
            }
        }
    ]
}
```

Reply:
This query requires querying the data pool directly, therefore your application needs to have the `data_pool:query` scope, please add this scope in console and try again.

https://www.propeldata.com/docs/control-access/api-scopes

Please note you will also need to have a tacosoft data pool in your account.